yunohost_project_organization.md

```diff
@@ -1,41 +1,41 @@
 # YunoHost project organisation
 
-## Document purpose
+## Purpose
 
-This document aims to allow contributors to feel legitimate in contributing to the YunoHost project together with collective feedback.
-The project is community-based and hasty decisions made by a restricted group of contributors can generate frustrations at a later stage.
-To address this issue, the proposed solution here is to ensure that decisions are taken collectively and that they are sufficiently thought out.
-An advisory council provides orientations for the evolution of the YunoHost project and special interest groups allow more efficient contribution in relation to each specific topic.
+Feel legitimized in contributing to the YunoHost project and part-take in the collective feedback.
+The project is community-based to avoid hasty decisions made by a restricted group of contributors generating frustrations at a later stage.
```

Comment:
```suggestion
Considering that the project is community-based, hasty decisions should be avoided so that everybody can participate and stay aware of what is happening.
```